recipelist.js

```diff
@@ -0,0 +1,61 @@
+// scripts.js
+
+// Sample recipe data
+const recipes = [
+    { id: 1, title: 'Pancakes', image: 'pancakes.jpg', link: 'pancakes.html' },
```

Comment:
The image location has not been specified for line 5, 6 and 7. The browser will attempt to find the image in the same folder as this file. Since all images are located in the images/ folder you would need to update it to reflect the correct location.
**NOTE:** I can't find **pancakes.jpg** in the images folder submitted by Sheree but there is a **pancakesdark.jpg**. Is that the file you were referring to?

Reply:
yeah i write the name of images as example ,  i want to put their recipes